test/micro/org/openjdk/bench/vm/compiler/VectorReductionFloatingMinMax.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2021, Huawei Technologies Co., Ltd. All rights reserved.
```

Comment:
Should each new source file also includes Oracle copyright?

Reply:
Fixed, thank you for mentioning this.

Reply:
> Should each new source file also include Oracle copyright?

I'm not sure.
But there are already some files  which don't contain Oracle copyright.

@dholmes-ora , what do you think of this question?
Thanks.